app/models/concerns/fog_extensions/google/server.rb

```diff
@@ -77,6 +76,13 @@ def associate_external_ip
         external_nat_present?
       end
 
+      # Added a getter method here as this attribute removed from fog-google
+      def network
```

Comment:
this implementation seems like there could be more networks  assigned to one machine through `interfaces`. What would matched better the foreman ideology. Or am I reading it wrong? :)

Reply:
+1 for multiple network support in case of GCE as well. 
As of now, it only supports single network selection. In case of EC2, it allows user to choose network -> subnet -> multiple security groups.

In a same way, GCE also have network -> subnets. There is one [feature request](https://projects.theforeman.org/issues/23366) to have subnet selection. I would suggest to consider this multiple network support under this feature implementation.

@ezr-ondrej, @tbrisker, any suggestions on this?

Reply:
not really familiar with gce or ec2, but my only suggestion is that where possible to try to have similar workflows across resources